bin/prune-test-files.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+set -ux
+
+# Synopsis:
+# Helpful for local testing as the UCM writes files as a result of the transcript process.
+
+exit_code=0
+
+# Iterate over all test directories
+for test_dir in tests/*; do
+    test_dir_path=$(realpath "${test_dir}")
+    results_file_path="${test_dir_path}/results.json"
+    test_loader_name="${test_dir_path}/.meta/testLoader.output.md"
```

Comment:
I don't know if this makes sense, but many test runners just .gitignore these files.

Reply:
> I might be the only person who uses the ucm-arm64 build 😆, I included it in case there was someone also struggling with the new mac chip format. I'll gitignore it and prune it from the git history. 

The prune test files script is just something I added for my local development process. I'll update the gitignore to exclude the files from git so no accidental check-ins of the output files end up happening.

Reply:
Ah okay, great!